rotkehlchen/server.py

```diff
@@ -54,7 +54,7 @@ def main(self) -> None:
         hub.exception_stream = None
         # we don't use threadpool much so go to 2 instead of default 10
         hub.threadpool_size = 2
-        hub.threadpool.maxsize = 2
+        hub.threadpool.maxsize = 3
```

Comment:
why do you need the extra thread here? We use all 2 and you need a third for this?

Reply:
I thought it was needed but seems that it doesn't affect after checking with the app. 2 seems fine